communication.py

```diff
@@ -73,53 +73,89 @@ def getData(self):
 
         # header = line[0] + line[1]
+        
         length = line[2]
         length = int.from_bytes(length, "little", signed=False)
 
-        takim_no = line[3] + line[4]
-        takim_no = int.from_bytes(takim_no, "little", signed=False)
-        takim_no = takim_no * 8
-
-        paket_no = line[5] + line[6]
+        paket_no = line[3] + line[4]
         paket_no = int.from_bytes(paket_no, "little", signed=False)
+
+        status = line[5]
+        status = int.from_bytes(status, "little", signed=False)
+
+        aras = line[6]
+        aras = int.from_bytes(aras, "little", signed=False)
+
         day = line[7]
         day = int.from_bytes(day, "little", signed=False)
+
         month = line[8]
         month = int.from_bytes(month, "little", signed=False)
+
         year = line[9]
         year = int.from_bytes(year, "little", signed=False)
+
         hour = line[10]
         hour = int.from_bytes(hour, "little", signed=False)
+
         minute = line[11]
         minute = int.from_bytes(minute, "little", signed=False)
+
         second = line[12]
         second = int.from_bytes(second, "little", signed=False)
 
         pressure_pl = line[13] + line[14] + line[15] + line[16]
         [pressure_pl] = struct.unpack("f", pressure_pl)
+
         pressure_car = line[17] + line[18] + line[19] + line[20]
         [pressure_car] = struct.unpack("f", pressure_car)
 
         height_pl = line[21] + line[22] + line[23] + line[24]
         [height_pl] = struct.unpack("f", height_pl)
+
         height_car = line[25] + line[26] + line[27] + line[28]
         [height_car] = struct.unpack("f", height_car)
+
         height_diff = line[29] + line[30] + line[31] + line[32]
         [height_diff] = struct.unpack("f", height_diff)
 
         speed = line[33] + line[34] + line[35] + line[36]
         [speed] = struct.unpack("f", speed)
+
         tempe = line[37] + line[38] + line[39] + line[40]
         [tempe] = struct.unpack("f", tempe)
+
         b_voltage = line[41] + line[42] + line[43] + line[44]
         [b_voltage] = struct.unpack("f", b_voltage)
 
         latitude_pl = line[45] + line[46] + line[47] + line[48]
         [latitude_pl] = struct.unpack("f", latitude_pl)
+
         longitude_pl = line[49] + line[50] + line[51] + line[52]
-        [longitude_pl] = struct.unpack("f", longitude_pl)
+        [longitude_pl] = struct.unpack("f", longitude_pl)          
+
         altitude_pl = line[53] + line[54] + line[55] + line[56]
         [altitude_pl] = struct.unpack("f", altitude_pl)
+        
+        yaw = line[57]+line[58]+line[59]+line[60]
+        [yaw] = struct.unpack("f", yaw)
+
+        roll = line[61]+line[62]+line[63]+line[64]
+        [roll] = struct.unpack("f", roll)
+
+        pitch = line[65]+line[66]+line[67]+line[68]
+        [pitch] = struct.unpack("f", pitch)
+
+        takim_no = line[69] + line[70]
+        takim_no = int.from_bytes(takim_no, "little", signed=False)
+        takim_no = takim_no * 8
```

Comment:
this part is written for team number at last year. that number was more bigger than 2 bytes and after received that, we were multiplying it with 8. (so it was sended as divided by 8). this part is not necessary for now.